apps/web/components/MailingListSignup/MailingListSignup.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useState, useEffect } from 'react'
```

Comment:
I wonder if it's possible to remove `/apps/web/screens/AboutPage/RenderForm.tsx` and replace the use of it inside `AboutPage` with this component instead since they are pretty much the same?

Reply:
We will be removing everything under `AboutPage` soon (:tm:) because the new Stafraent Island page has replaced it.